Task 1/Return subsequences of a string.cpp

```diff
@@ -0,0 +1,34 @@
+#include <iostream>
+using namespace std;
```

Comment:
If you want your code to sit in Task 1, you would also need a `question.md` file. The folder structure has been mentioned in the `README.md` file. You can alternatively move this to the `Task 3` folder, which doesn't require you to have a `question.md` file. 

Make sure to make a folder inside these task folders called `Return subsequences of a string` inside which you can put your `.cpp` file.